core/src/main/java/org/infinispan/factories/EmptyConstructorFactory.java

```diff
@@ -27,31 +23,26 @@
  * @since 4.0
  */
 
-@DefaultFactoryFor(classes = {InboundInvocationHandler.class, RemoteCommandsFactory.class, ExternalizerTable.class,
-                              BackupReceiverRepository.class, CancellationService.class, TimeService.class})
+@DefaultFactoryFor(classes = {BackupReceiverRepository.class, CancellationService.class, ExternalizerTable.class,
+                              InboundInvocationHandler.class, RemoteCommandsFactory.class, TimeService.class})
```

Comment:
In the above lines I sorted the fields because I had a hard time seeing which parameters were there.